core/bin/zksync_api/src/fee_ticker/ticker_api/mod.rs

```diff
@@ -316,3 +315,91 @@ impl<T: TokenPriceAPI + Send + Sync> FeeTickerAPI for TickerApi<T> {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use bigdecimal::ToPrimitive;
+    use std::env;
+    use zksync_types::{Address, Token, TokenId, TokenKind, TokenPrice};
+
+    #[tokio::test]
+    async fn should_return_one_for_rdoc() {
+        const DATABASE_URL: &str = "postgres://postgres@localhost/plasma";
+        const RDOC_SYMBOL: &str = "RDOC";
+        const RDOC_VALUE: u32 = 1;
+
+        env::set_var("DATABASE_URL", DATABASE_URL);
+
+        struct FakeTickerApi;
+
+        #[async_trait::async_trait]
+        impl TokenPriceAPI for FakeTickerApi {
+            async fn get_price(&self, _token: &Token) -> Result<TokenPrice, PriceError> {
+                Err(PriceError::token_not_found("Wrong token"))
+            }
+        }
+
+        let token = TokenLike::Symbol(String::from(RDOC_SYMBOL));
+
+        let connection_pool = ConnectionPool::new(Some(1));
+        let ticker_api = TickerApi::new(connection_pool, FakeTickerApi);
+
+        let actual_qoute = FeeTickerAPI::get_last_quote(&ticker_api, token)
+            .await
+            .unwrap();
+
+        assert_eq!(actual_qoute.usd_price.to_u32().unwrap(), RDOC_VALUE);
+    }
+
+    #[tokio::test]
+    async fn should_return_value_from_cache() {
+        const DATABASE_URL: &str = "postgres://postgres@localhost/plasma";
+        const TEST_TOKEN_SYMBOL: &str = "TEST";
+        const TOKEN_VALUE: u32 = 5;
+
+        env::set_var("DATABASE_URL", DATABASE_URL);
```

Comment:
Im not sure about this, I got the same error but just when tried to run it directly, but by running it with the zk command, it works properly.